collect_app/src/main/java/org/odk/collect/android/preferences/AdminSharedPreferences.java

```diff
@@ -61,7 +61,7 @@ public void reset(String key) {
 
     public void save(String key, Object value) {
         editor = sharedPreferences.edit();
-        if (value == null || value == "" || value instanceof String) {
```

Comment:
I argue `value == ""` in this expression is redundant, `value == ""` returns true only and only if `value` is instance of `String` , in this case `value instanceof String` also returns which short-circuit the expression.
There is no scenario where this expression returns true solely because of `value == ""`

Reply:
Yes, thanks -- good change.